src/metadatas.rs

```diff
@@ -84,8 +84,14 @@ pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
             .unique()
             .collect(),
         issues_count: std::collections::BTreeMap::new(),
-        has_fares: gtfs.fare_attributes.is_some(),
-        has_shapes: gtfs.shapes.is_some(),
+        has_fares: match &gtfs.fare_attributes {
+            Some(Ok(fa)) => fa.len() > 0,
+            _ => false,
+        },
+        has_shapes: match &gtfs.shapes {
+            Some(Ok(s)) => s.len() > 0,
```

Comment:
I think it's a bit better to check this with https://doc.rust-lang.org/std/vec/struct.Vec.html#method.is_empty